specs/mediawiki_v1_graphoid.yaml

```diff
@@ -24,7 +24,7 @@ paths:
           required: true
         - name: revision
           in: path
-          description: The page revision
+          description: The page revision, or 0 if unknown
```

Comment:
I should have said precise perhaps. It's good to be explicit about which revision is returned when 0 is passed in.